libcfnet/server_code.c

```diff
@@ -7,6 +7,7 @@
 #include <systype.h>                    // CLASSTEXT
 #include <signals.h>                    // GetSignalPipe
 #include <cleanup.h>                    // DoCleanupAndExit
+#include <ctype.h>
```

Comment:
Nitpick, but I think it's useful to add the comments for why you added the include, similar to lines above.